src/test/resources/features/install_candidate.feature

```diff
@@ -17,7 +17,7 @@ Feature: Install Candidate
 
 	Scenario: Install a specific Candidate and set to default
 		Given the system is bootstrapped
-		And the candidate "grails" version "1.3.9" is available for download
+		And the candidate "grails" version "1.3.9" is available for download with checksum "1f9234c8e622ec46d33883ea45b39ede768b92d478fe08f6952548247f7fbb65" using algorithm "SHA-256"
```

Comment:
I'd prefer if we introduce a new feature file for the checksums. I don't think we should pollute this feature with such cross-cutting concerns as checksumming. wdyt?

As far as I can tell, these tests will still pass if the headers aren't present right?

Reply:
done